lib/pgbouncerhero/database.rb

```diff
@@ -17,7 +17,7 @@ def name
     end
 
     def connection
-      connection_model.new(host, port, user, password, dbname).connection
+      @connection ||= connection_model.new(host, port, user, password, dbname).connection
```

Comment:
I believe calling `connection` checks a connection out of the pool. Caching just the model would be safer.